pkg/broker/ensure.go

```diff
@@ -38,7 +38,7 @@ func Ensure(config *rest.Config) error {
 		return fmt.Errorf("error setting up the engine requirements: %s", err)
 	}
 
-	err = lighthouse.Ensure(config)
+	_, err = lighthouse.EnsureMcsCRD(config)
```

Comment:
That’s a shame, I thought the whole point of the `Ensure` pattern was that it could be blindly used (largely)...

Reply:
yeah, but... we make a difference between what we "ensure" on the broker, and what we ensure on the operator... I needed a difference for that... better ideas are welcome.

Reply:
We could deploy all everywhere though, and not care. Which is what we would have in the case of the olm, right?

Reply:
We can use isBroker flag to differentiate betwen data and broker. We may not need ServiceExport CR on broker, but no harm in installing CRD everywhere.

Reply:
It's what will happen when we people setups the broker through olm...